app/lib/dfe/reference_data/degrees.rb

```diff
@@ -1,730 +1,669 @@
+            'qualification' => '181eb0ae-eeb8-487b-9c6a-a4ee3e1907b6',
             'dqt_id' => '656a5652-c197-e711-80d8-005056ac45bb',
             'hesa_itt_code' => '305' },
           '676a5652-c197-e711-80d8-005056ac45bb' =>
           { 'priority' => 1,
-            'name' => 'Doctor of Philosophy', # This is rarely actually in philosophy, so I'm leaving the topic nil
+            'name' => 'Doctor of Philosophy',
             'abbreviation' => 'PhD',
             'synonyms' => [],
-            'level' => '8',
-            'topic' => nil,
+            'qualification' => '181eb0ae-eeb8-487b-9c6a-a4ee3e1907b6',
             'dqt_id' => '676a5652-c197-e711-80d8-005056ac45bb',
             'hesa_itt_code' => '306' },
           '03d6b7af-499c-49e3-96cc-e63f9beda6e5' =>
           { 'priority' => 1,
             'name' => 'Doctor of Education',
             'abbreviation' => 'EdD',
             'synonyms' => [],
-            'level' => '8',
-            'topic' => 'Ed',
+            'qualification' => '181eb0ae-eeb8-487b-9c6a-a4ee3e1907b6',
             'dqt_id' => nil,
             'hesa_itt_code' => nil } },
-          '0584565a-1c98-4c1d-ae64-c241542c0879' =>
-          { 'priority' => 1,
-            'name' => 'First Degree',
-            'abbreviation' => nil,
-            'synonyms' => [],
-            'level' => nil,
-            'topic' => nil,
-            'dqt_id' => nil,
-            'hesa_itt_code' => '400' },
-          'fdafdcd7-5f21-4363-b7d5-c1f44a852af1' =>
-          { 'priority' => 1,
-            'name' => 'Higher Degree',
-            'abbreviation' => nil,
-            'synonyms' => [],
-            'level' => nil,
-            'topic' => nil,
-            'dqt_id' => nil,
-            'hesa_itt_code' => '401' },
-          '03c4fa67-345e-4d09-8e9b-68c36a450947' =>
-          { 'priority' => 1,
-            'name' => 'Degree equivalent',
-            'abbreviation' => nil,
-            'synonyms' => [],
-            'level' => nil,
-            'topic' => nil,
-            'dqt_id' => nil,
-            'hesa_itt_code' => '402' },
+        # "Generic" HESA codes
+        '0584565a-1c98-4c1d-ae64-c241542c0879' =>
+        { 'priority' => 1,
+          'name' => 'First Degree',
+          'abbreviation' => nil,
+          'synonyms' => [],
+          'qualification' => nil,
+          'dqt_id' => nil,
+          'hesa_itt_code' => '400' },
+        'fdafdcd7-5f21-4363-b7d5-c1f44a852af1' =>
+        { 'priority' => 1,
+          'name' => 'Higher Degree',
+          'abbreviation' => nil,
+          'synonyms' => [],
+          'qualification' => nil,
+          'dqt_id' => nil,
+          'hesa_itt_code' => '401' },
+        '03c4fa67-345e-4d09-8e9b-68c36a450947' =>
+        { 'priority' => 1,
+          'name' => 'Degree equivalent',
+          'abbreviation' => nil,
+          'synonyms' => [],
+          'qualification' => nil,
+          'dqt_id' => nil,
+          'hesa_itt_code' => '402' },
```

Comment:
Should we have these in a separate list, as they're not really qualifications, but just generic categories of qualifications? In some contexts we might want to include them, eg Register, but in others we might want to filter them out and let people enter their actual qualification as free text if it's missing from the list, eg Apply.

Reply:
Done, and a joined list provided for users who need the lot